src/Components/ProductList.js

```diff
@@ -0,0 +1,17 @@
+import Product from "./Product";
+
+function ProductList(props) {
+    return ( <>{props.items.map((individualItem) => {
```

Comment:
Line 4 - '(' and '<>' could be put onto two new lines so the code is easier to read.